private/extensions/maven.bzl

```diff
@@ -170,6 +171,7 @@ def _maven_impl(mctx):
     # - strict_visibility_value: a string list. Build will fail is duplicated and different.
     # - use_starlark_android_rules: bool. A logical OR over all `use_starlark_android_rules` for all `install` tags with the same name.
     # - version_conflict_policy: string. Fails build if different and not a default.
+    # - ignore_empty_files: Treat jars that are empty as if they were not found.
```

Comment:
Do you mind adding something to the README.md for this option?

Reply:
Sure, I could do that.

Reply:
Done.